envs/example/ci-full-rhel/group_vars/all.yml

```diff
@@ -9,9 +9,12 @@ state_path_base: /opt/stack/data
 
 validate_certs: false
 
-redhat:
-  user: XXXXXXX
-  pass: XXXXXXX
+rhn_subscription:
+  username: XXXXXXX
+  password: XXXXXXX
+  pool_names: []
```

Comment:
does this get used anywhere ?  I see pool_names_regex but no other mention of pool_names